### PR TITLE
Misc: Use argparse

### PR DESCRIPTION
**Pull request by Dietr1ch**
My take at #70.

I tried using `black` for formatting, but things are not in a great shape now, so I just tried to match the format on the lines I had to change.

This looks correct to me, but I haven't been able to run the converted model yet, so I can't verify. I was able to convert it just fine, but got some tensor dimension mismatch later.

### CODE REVIEW

File: convert-pth-to-ggml.py [hunk 1]
```diff
@@ -22,20 +22,34 @@
...
+                        type=str,
+                        required=True,
+                        help="Model to write")
+ARGS = ARG_PARSER.parse_args()
+
+FTYPE_IDX = -1
+if ARGS.ftype == "f16":
+    FTYPE_IDX = 1
+elif ARGS.ftype == "f32":
+    FTYPE_IDX = 0
```
Comment (j-f1) [inline convert-pth-to-ggml.py:49]: small suggestion:
```suggestion
FTYPE_IDX = {"f16": 1, "f32": 0}[ARGS.ftype]
```

Comment (Dietr1ch) [reply convert-pth-to-ggml.py]: That's neat. I wanted to take another stab at it, but was stuck the error case (before I noticed I can use choose+require in argparse to guarantee the selection is total)

**Comment by ggerganov** [comment]
Hi, thanks for contributing!
There is no need to add extra dependencies for such simple things, such as argument parsing.
The current way is good enough. It can be improved by adding comments and better help description, etc.

**Comment by leszekhanusz** [comment]
argparse is part of the Python standard library, so this PR does not add any new dependency.

**Pull request closed without merging**